indexers/keyvalue/RedisDBIndexer/__init__.py

```diff
@@ -51,13 +52,15 @@ def query(self, key: int, *args, **kwargs) -> Optional[bytes]:
             return None
 
         if len(results) > 1:
-            self.logger.warning(f'More than 1 element retrieved from Redis with matching key {key}. Will return first...')
+            self.logger.warning(
+                f'More than 1 element retrieved from Redis with matching key {key}. Will return first...')
         return results[0]['values']
 
     def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
-        """Add a JSON-friendly object to the indexer
-        :param keys: keys to be added
-        :param values: values to be added
+        """Add JSON-friendly serialized documents to the index.
+
+        :param keys: document ids
+        :param values: JSON-friendly serialized documents
```

Comment:
is correct with the type definition?

Reply:
What I mean is that the document is serialized, i.e., in byte form. Also it is required to be in a json friendly format after deserialization.

Reply:
Okey, good thanks